app/src/main/res/raw/segment_postprocess_shader.glsl

```diff
@@ -7,7 +7,7 @@ uniform sampler2D framebuffer;
 void main() {
     vec2 uv = gl_FragCoord.xy / resolution;
 
```

Comment:
Simplify and add a comment explaining this.

Reply:
fix

Reply:
ed